ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockHelixCluster.java

```diff
@@ -206,6 +206,14 @@ InstanceConfig getInstanceConfig(String instanceName) {
     return instanceConfig;
   }
 
+  List<InstanceConfig> getAllInstanceConfigs() {
```

Comment:
This method is used in next PR, right?

Reply:
Yes, this is needed for the unit tests, which is in progress. I just put this up, so the review can start, since we seemed to need this quickly.